### ISSUE TITLE
confusing warning if use `nimbleHMC` (or `buildHMC`) and not all nodes are sampled

### ISSUE THREAD

**Issue by paciorek** [nimble-dev/nimbleHMC#31]
```
  [Warning] No samplers assigned for 4 nodes, use conf$getUnsampledNodes() for node names.
```

But `conf` does not exist in this usage.

**Comment by danielturek** [comment]
@paciorek I don't think that should ever be the case.  Unless the underlying logic is faulty, `buildHMC` should always assign samplers to all nodes, and this warning would never be printed.

Is there a case where you saw this happen?  An instance where the code:
```
buildHMC(model_object)
```
issued this warning?

My apologies if I'm overlooking some case, or don't understand the question.

**Comment by paciorek** [comment]
`buildHMC` takes a `nodes` arg, so isn't it plausible a user could only assign the sampler to some nodes? Happy to make a reprex, but not sure it's needed.

**Comment by danielturek** [comment]
Fixed. The part about `conf$getUnsampledNodes()` is now omitted from that branch of messaging.
